src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -286,6 +286,10 @@ async def genomic_to_transcript_exon_coordinates(
         )
         if start is None and end is None:
             return self._return_warnings(resp, "Must provide either `start` or `end`")
+        if chromosome is None and alt_ac is None:
+            return self._return_warnings(
+                resp, "Must provide either `chromosome` or `alt_ac`"
+            )
```

Comment:
This may need to include transcript and gene (which is not included in docstring). Didn't fully go through to check this

Reply:
yeah, I can clean that up and check for gene/transcript in a subsequent PR